keps/sig-apps/1847-autoremove-statefulset-pvcs/README.md

```diff
@@ -100,17 +103,22 @@ delete the PVCs created by the controller from the StatefulSet's VolumeClaimTemp
 
 The following changes are required:
 
-1. Add `PersistentVolumeClaimDeletePolicy` to the StatefulSet spec with the following policies.
+1. Add a `PersistentVolumeClaimDeletePolicy` to the StatefulSet spec with the following policies.
    * `Retain` - this is the default policy and is considered in cases where no policy is
        specified. This would be the existing behaviour - when a StatefulSet is deleted, no
        action is taken with respect to the PVCs created by the StatefulSet.
-   * `DeleteOnStatefulSetDeletion` - PVCs corresponding to the StatefulSet are deleted when StatefulSet
-       themselves get deleted. When Pods are deleted as part of a scale down, PVCs are not
-       deleted. Thus there may be PVCs owned by the StatefulSet that are not attached to a Pod.
-   * `DeleteOnScaledown` - When a pod is deleted on scale down, the corresponding PVC is deleted as well. 
-       On a scale down followed by a scale up, will wait until the old PVC for the removed Pod is deleted and ensure 
-       that the PVC used is a freshly created one. This policy also implies the
-       former, that is, PVCs will also be deleted when the StatefulSet is deleted.
+   * `DeleteOnStatefulSetDeletion` - PVCs corresponding to the StatefulSet are deleted
```

Comment:
Should this option also have an "Only" suffix?

Reply:
Good point, done.

Reply:
Done